zoofilter/01.basics/03.integration/docs.md

```diff
@@ -3,3 +3,156 @@ title: Integration
 taxonomy:
     category: docs
 ---
+
+## Search form
+
+ZOOfilter Search form is displayed by the *ZOOfilter Module* and controlled by the *ZOOfilter Plugin*. Currently is limited to search through one App and Type at the time.
+
+The Form inputs are composed by the Elements in the ZOOfilter layouts.
+
+To configure the search form please follow the next instructions:
+
+1. Go to the ZOO/gearwheel tab/<your app>
+2. Open the *ZOOfilter Search* Default layout for the Type you would like to create the Form for. 
+3. Drag & Drop the elements you would like the Search Form to be composed by.
+
+For detailed information please review the [ZOO Positions Assignments doc](http://yootheme.com/zoo/documentation/advanced/assign-elements-to-layout-positions).
+
+Notice that the render layouts and params are specific for the form purpose. Only the searchable and Elements with a group are compatible with this layout. 
+
+### Fields
+
+ZOOfilter Form fields are composed by the Elements for whom ZOOfilter will use specific group layouts. Those layouts are stored in the **modules/mod_zoofilter/tmpl/_elements** folder, where you can even store your custom layouts.
```

Comment:
the path in the translation doc also should be in `administrator/language/{LANG}` instead of **administrator/language/{LANG}**? This file is the same for other extensions